src/things/comment.rs

```diff
@@ -61,6 +61,19 @@ impl Shred for Comment {
             return;
         }
 
+        if config.prevent_comment_deletion {
+            debug!("Skipping DELETION due to `prevent_comment_deletion` filter ({})", config.prevent_comment_deletion);
+            match &config.gdpr_export_dir {
+                Some(_) => {}
+                None => {
```

Comment:
If you want to just handle a `None` case you can do `if config.gdpr_export_dir.is_none()`. You can run `cargo clippy` and it should help with stuff like this (if you're interested).